hera_cal/lst_stack/config.py

```diff
@@ -219,164 +225,28 @@ def get_all_unflagged_baselines(
     return sorted(all_baselines), sorted(all_pols)
 
 
-def config_lst_bin_files(
-    data_files: list[list[str | FastUVH5Meta]],
-    dlst: float | None = None,
-    atol: float = 1e-10,
-    lst_start: float | None = None,
-    lst_width: float = 2 * np.pi,
-    blts_are_rectangular: bool | None = None,
-    time_axis_faster_than_bls: bool | None = None,
-    ntimes_per_file: int | None = None,
-    jd_regex: str = r"zen\.(\d+\.\d+)\.",
-):
-    """
-    Configure data for LST binning.
-
-    Make a 24 hour lst grid, starting LST and output files given
-    input data files and LSTbin params.
+def _subsorted_list(x: Sequence[Sequence[Any]]) -> list[list[Any]]:
+    return [sorted(y) for y in x]
 
-    Parameters
-    ----------
-    data_files : list of lists
-        nested set of lists, with each nested list containing paths to
-        data files from a particular night. Frequency axis of each file must be identical.
-    dlst : float
-        LST bin width. If None, will get this from the first file in data_files.
-    atol : float
-        absolute tolerance for LST bin float comparison
-    lst_start : float
-        starting LST for binner as it sweeps from lst_start to lst_start + 2pi.
-        Default is first LST of the first file of the first night.
-    lst_width : float
-        How much LST to bin.
-    ntimes_per_file : int
-        number of LST bins in a single output file
 
-    Returns
-    -------
-    lst_grid : float ndarray holding LST bin centers.
-    matched_files : list of lists of files, one list for each output file.
+@attrs.define(slots=False, frozen=True)
+class LSTBinConfiguration:
```

Comment:
Perhaps change to `LSTBinConfigurator` for clarity

Reply:
Good suggestion. Done.